README.md

```diff
@@ -47,18 +47,29 @@ make git_run
 
 Note : Both docker and git pulls latest version of openapi generator.
 
-## Adding complex workflows:
-Complex workflows requiring multiple granular, autogenerated apis can be implemented as part of Operator classes. These classes are included in the path _equinix-openapi-metal/src/main/java/com/equinix/workflow_ when building the sdk.
+## Contribution guidelines
+### Patching oas3.0 spec
+1. Make changes in ``spec/oas3.patched.json``.
+2. Create a patch file in metal-java:
+   ```
+   cd spec
+   git diff oas3.patched.json > ../patches/spec.fetched.json/<patchfilename>
```

Comment:
Increasing the context on the patches will help to identify the source. I've had problems in the past where the patches were fuzz applied to an adjacent object with a similar structure. `git diff -U10`, for example.

Reply:
Mentioned in https://github.com/equinix-labs/metal-java/pull/48/commits/c7f923d9729a6b31d5e25e8a93fb2c10c28fd4d5